src/scribe.js

```diff
@@ -191,7 +191,7 @@ define([
 
   Scribe.prototype.getContent = function () {
     // Remove bogus BR element for Firefox — see explanation in BR mode files.
-    return this._htmlFormatterFactory.formatForExport(this.getHTML().replace(/<br>$/, ''));
+    return this._htmlFormatterFactory.formatForExport(this.getHTML().replace(/<br>$/, '').replace(/<br>/g, '<br/>'));
```

Comment:
I thought that self-closing `/` was only necessary in XHTML? this is for our own parsing, right?

Reply:
Yeah, that's right. Makes a difference in the truncation lib that I'm using. Self-closing void elements are allowed, but not required, in HTML5 you are right: https://html.spec.whatwg.org/multipage/syntax.html#start-tags. I'll update the the PR text.